plasma/plasma-mvp/morevp-simplified.md

```diff
@@ -0,0 +1,87 @@
+# More Viable Plasma
+
+## TL;DR
+
+This document provides a largely simplified specification for More Viable Plasma, a version of [Minimal Viable Plasma](https://ethresear.ch/t/minimal-viable-plasma/426) (“Plasma MVP”) that removes the need for confirmation signatures.
+
+## Background
+
+Transactions in Plasma MVP are processed in time-wise order.
+The “exit priority” of any UTXO is determined by the block in which the transaction was included, then the index of the transaction within that block, and finally the index of the output in the list of transaction outputs (“output-age priority”).
+Plasma MVP requires the use of confirmation signatures to prevent the operator from including a valid transaction time-wise after an invalid transaction.
```

Comment:
Suggestion let's stop referencing the obsolete (soon-to-be-forgotten) confirm-sigs, and focus on the problem at hand (in-flight transactions exitability). If this is intended to become part of spec, confirm-sigs are just confusing and irritating to read about for bazilionth time

Reply:
The point of this section is to give background and to motivate the rest of the design. It's important that people understand not only the design but the thought processes that led to that design.